### PR TITLE
Fixes #34150 -  align secondary tabs in host page

### PR DESCRIPTION
**Pull request by amirfefer**
following https://github.com/theforeman/foreman/pull/8996

before:
Screen Shot 2021-12-14 at 18 40 26

after:
Screen Shot 2021-12-14 at 19 52 45

### CODE REVIEW

**Comment by Andrewgdewar** [comment]
Looking good! I would Ack but I'm an imposter!

File: webpack/components/AnsibleHostDetail/AnsibleHostDetail.js [hunk 1]
```diff
@@ -21,6 +21,7 @@ const AnsibleHostDetail = ({
       {response?.id && (
         <>
           <Tabs
+            className="host-details-tabs"
```
Comment (ezr-ondrej) [inline webpack/components/AnsibleHostDetail/AnsibleHostDetail.js:24]: it is quite generic className, give we are not scoping the css, should we give it bit more specific name (perhaps `ansible-host-details-tabs`?

**Comment by amirfefer** [comment]
@ezr-ondrej, done :)

**Comment by ezr-ondrej** [comment]
> Looking good! I would Ack but I'm an imposter!

Btw @Andrewgdewar imposters can also ack :) it's always nice to see that someone agrees with the solution. It gives the maintainers more confidence while merging :+1:

**Review by ezr-ondrej — APPROVED** [review]
Thanks @amirfefer ! :+1:

**Comment by ezr-ondrej** [comment]
CP 7.0: fab805d

**Pull request merged**